### PR TITLE
Various cleanups around the codebase

### PR DESCRIPTION
**Pull request by vagababov**
- unindent the else after return
- make things private that are not used anywhere
- rearrange params
- etc

/assign mattmoor

### CODE REVIEW

File: testutils/clustermanager/e2e-tests/boskos/fake/fake.go [hunk 1]
```diff
@@ -65,9 +65,8 @@ func (c *FakeBoskosClient) ReleaseGKEProject(name string) error {
 				res.Owner = ""
 				res.State = boskoscommon.Free
 				return nil
-			} else {
-				return fmt.Errorf("Got owner: '%s', expect owner: '%s'", res.Owner, owner)
 			}
+			return fmt.Errorf("Got owner: '%s', expect owner: '%s'", res.Owner, owner)
```
Comment (mattmoor) [inline testutils/clustermanager/e2e-tests/boskos/fake/fake.go:69]:
```suggestion
			return fmt.Errorf("Got owner: %q, expect owner: %q", res.Owner, owner)
```

**Comment by mattmoor** [comment]
/hold

for the comment

File: testutils/clustermanager/e2e-tests/boskos/fake/fake.go (hunk 1 above)
Comment (vagababov) [reply testutils/clustermanager/e2e-tests/boskos/fake/fake.go]: Done and also on the next line.

**Comment by vagababov** [comment]
Duh, I need to fix this test
/retest

**Pull request merged**